src/matching.rs

```diff
@@ -6,7 +6,9 @@ use std::convert::TryInto;
 #[derive(Serialize, Deserialize, Debug)]
 pub(crate) struct BaoSymbol {
     pub(crate) name: String,
+    #[serde(default)]
```

Comment:
What's the meaning of this?

Reply:
thats my quick fix for getting json files produced by fakepdb to load into the application